projects/core/src/accordion/accordion-header.element.scss

```
@@ -8,11 +8,12 @@
 @import './../styles/mixins/mixins';
 
 /*
- * The variables `--box-shadow`, `--icon-visibility`, and `--icon-margin` are
+ * The variables '--box-shadow', '--icon-visibility', and '--icon-margin' are
  * included to provide users with the flexibility to extend the accordion
  * component, enabling them to create a stepper component that aligns with the
  * style of the ng-clarity stepper.
- */
+*/
+
```

```suggestion
 */
```

This leftover change should be reverted as well.